dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -2078,7 +2078,7 @@ public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop
                 resetLogOffsets();
                 clearMessageCounters();
             }
-            if (GLOBAL_DEBUG_JAVA2SECURITY)
+            if (GLOBAL_JAVA2SECURITY)
```

Comment:
What's this change about?

Reply:
This change mans the `ACEScanner` tool will run by default locally.  What it does is it scans through the server logs on server stop and rolls up all of the unique j2sec issues into a report that ends up in the autoFVT dir.  It's mentioned in a bit more detail here: 
https://github.com/OpenLiberty/open-liberty/wiki/FAT-tests#java-2-security-testing

But now that I look at this again, I'm going to make a slight tweak and have the `ACEScanner` run for either condition (GLOBAL_JAVA2SECURITY OR GLOBAL_DEBUG_JAVA2SECURITY)